Libraries/src/Amazon.Lambda.AmazonMQEvents/Amazon.Lambda.AmazonMQEvents.csproj

```
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\..\buildtools\common.props" />
+	
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
+    <Description>Amazon Lambda .NET Core support - AmazonMQEvents package.</Description>
+    <AssemblyTitle>Amazon.Lambda.AmazonMQEvents</AssemblyTitle>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <AssemblyName>Amazon.Lambda.AmazonMQEvents</AssemblyName>
```

```suggestion
    <AssemblyName>Amazon.Lambda.MQEvents</AssemblyName>
```

I don't see yet another Amazon prefix before event names, this should just be `MQEevents`.

---

The reason I prefixed it with `Amazon` is that the documentation page https://aws.amazon.com/amazon-mq/?amazon-mq.sort-by=item.additionalFields.postDateTime&amazon-mq.sort-order=desc mentions it as `Amazon MQ`. Please advise if we should still rename it to `MQEvents`. If yes, I would need to submit new revision of pipeline CR to update it with new name.

---

@ganeshnj I have renamed the package to `Amazon.Lambda.MQEvents`. Please review.

---

Awesome thanks for making the change.